src/node_api.h

```diff
@@ -166,6 +170,18 @@ NAPI_EXTERN napi_status napi_get_value_string_length(napi_env env,
                                                      napi_value value,
                                                      int* result);
 
+// Gets the number of BYTES in the LATIN-1 encoded representation of the string.
+NAPI_EXTERN napi_status napi_get_value_string_latin1_length(napi_env env,
```

Comment:
I'm not sure we need this. Since latin1 is a one-byte encoding, the byte length is the same as the character length, so `napi_get_value_string_length` could be used instead.